src/components/SpaceProposalNFTSidebar.vue

```diff
@@ -0,0 +1,133 @@
+<script setup lang="ts">
+import { ExtendedSpace, Proposal } from '@/helpers/interfaces';
+import { MINT_CURRENCY } from '@/helpers/nftClaimer';
+
+const props = defineProps<{
+  space: ExtendedSpace;
+  proposal: Proposal;
+}>();
+
+useNFTClaimer(props.space, props.proposal);
+const { getContractInfo, getCollectionInfo } = useNFTClaimerStorage();
+const { web3Account } = useWeb3();
+const { modalAccountOpen } = useModal();
+const { formatNumber } = useIntl();
+// TODO enable in production
+// const { isSpaceController } = useSpaceController();
+const isSpaceController = true;
+
+const isModalMintOpen = ref(false);
+const isModalExploreOpen = ref(false);
+const shouldReopenMintModal = ref(false);
+
+function reopenMintModal() {
+  if (shouldReopenMintModal.value) {
+    isModalMintOpen.value = true;
+    shouldReopenMintModal.value = false;
+  }
+}
+
+watch(
+  () => web3Account.value,
+  () => reopenMintModal()
+);
+
+const contractInfo = computed(() => {
+  return getContractInfo(props.space.id);
+});
+
+const collectionInfo = computed(() => {
+  return getCollectionInfo(props.space.id, props.proposal.id);
+});
+</script>
+
+<template>
+  <template v-if="collectionInfo">
+    <BaseBlock v-if="contractInfo" :title="$t('SnapIt!')" :slim="true">
+      <div class="flex flex-col space-y-4 p-4">
+        <div class="group flex flex-row gap-4">
+          <NFTClaimerLogo class="shrink-0" />
+          <span class="text-sm leading-tight">
+            Mint your NFT now to show your support for this proposal.
+            <a href="https://docs.snapshot.org/">Learn more</a>
+          </span>
+        </div>
+
+        <SpaceProposalNFTProgress
+          :max-supply="collectionInfo.maxSupply"
+          :supply="collectionInfo.mintCount"
+          show-info
+          class="cursor-pointer"
+          tabindex="0"
+          title="View list of minted NFTs"
+          @click="isModalExploreOpen = true"
+        >
+          <template #secondary>
+            <i-ho-arrow-top-right-on-square class="ml-2 mt-2 text-xs" />
+          </template>
+        </SpaceProposalNFTProgress>
+      </div>
+      <div
+        class="flex flex-row content-center items-center justify-between border-t px-4 py-3"
+      >
+        <div class="flex flex-col">
+          <span>Mint price</span>
+          <span class="text-base font-bold text-skin-link">
+            {{ formatNumber(collectionInfo.formattedMintPrice) }}
```

Comment:
I put mint price `0.00001` it shows `0` instead.

Reply:
<img width="416" alt="image" src="https://github.com/snapshot-labs/snapshot/assets/51686767/284bd9fa-fc31-4f78-910d-9ce0777c99c4">

Reply:
Would be great to also see the mint price in USD

Reply:
> Would be great to also see the mint price in USD

It's shown in the mint modal. We could also show it here, but feels it'll overcrowd, or dissuade people when they see fiat

Reply:
> I put mint price 0.00001 it shows 0 instead.

Is there a function to pretty print ETH value ?

Reply:
yarn hardhat node

Reply:
Can't we use `formatCompactNumber` like we do in `SpaceProposalVotesListItem` for voting power?

Reply:
done